canonical-web-and-design/jaas-dashboard
# Add model counts as chips

## Done

- Add model counts as chips
- Switched StatusStrip to TS
- Moved utils.js into it's own folder and started to add tests

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check models and model details views across breakpoints and verify chips look mint

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1632
